app/containers/ViewQuestion/saga.js

```diff
@@ -711,7 +711,7 @@ export function* downVoteWorker({
     yield call(isAuthorized);
 
     const isOwnItem =
-      questionData.author === profileInfo.user ||
+      questionData.author.user === profileInfo.user ||
```

Comment:
on 781 line you have the same check condition, extract to some function, it should be controlled via one place